docs/en/setup/service-agent/java-agent/Supported-list.md

```diff
@@ -25,6 +25,7 @@ metrics based on the tracing data.
   * [Jersey REST framework](https://github.com/eclipse-ee4j/jersey) 2.x -> 3.x
   * [Grizzly](https://github.com/eclipse-ee4j/grizzly) 2.3.x -> 4.x
   * [WebSphere Liberty](https://github.com/OpenLiberty/open-liberty) 23.x
+  * [Netty HTTP](https://github.com/netty/netty) 4.1.x
```

Comment:
```suggestion
  * [Netty HTTP](https://github.com/netty/netty) 4.1.x (Optional¹)
```

I checked your test scenario, this plugin seems to work for both client and server of Netty? Am I right? If so, I feel it is better to put it inside `RPC` catalog.

Reply:
And `optional` suffix is required as it is not activated by default.

Reply:
> I checked your test scenario, this plugin seems to work for both client and server of Netty? Am I right? If so, I feel it is better to put it inside `RPC` catalog.
Yes ,it works on the both side.

Reply:
> I checked your test scenario, this plugin seems to work for both client and server of Netty? Am I right? If so, I feel it is better to put it inside `RPC` catalog.

Yes,it works on the both side

Reply:
> (Optional¹)

![image](https://github.com/apache/skywalking-java/assets/96932958/488bc533-15b0-456e-bcdc-3cabcfd39c44)
Is it ok?

Reply:
It is Optional 2. Optional 1 means having a license incompitable issue.

Reply:
The descriptions of Optional 1 and 2 are at the bottom of the doc file.

Reply:
> incompitable

I got it